app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,66 @@
+class RecipesController < ApplicationController
+  before_action :find_recipe, only: %i[show destroy update]
+  before_action :authenticate_user!
+
+  def index
+    @recipes = Recipe.includes(:user).where(user_id: current_user.id)
+    puts "El usuario es #{current_user.id}"
+  end
+
+  def show
+    unless @recipe.public || current_user == @recipe.user
+      redirect_to root_path, alert: 'You cannot access it.'
+      return
+    end
+    @recipe_foods = @recipe.recipe_food
+  end
+
+  def new
+    @recipe = Recipe.new
+  end
+
+  def create
+    @recipe = Recipe.new(recipe_params)
+    @recipe.user = current_user
+    puts params.inspect
+    if @recipe.save
+      redirect_to recipe_path(@recipe)
+    else
+      render :new
+    end
+  end
```

Comment:
- [ ] Dear Team, Good job with the project. The Issue I noticed on your recipe app is that I could not create a recipe. Every time I try, it keeps throwing an error, Pease make sure it's possible to add a recipe to your app. 

### The issue 

https://github.com/ulises2607/rails-recipe-app/assets/69077061/c5bce8e8-9e3d-45fa-8b1e-265c7d3915ae

#### While doing this, Please keep in mind the following feature, which I could not check now.
- You can remove a recipe item only if you're the owner
- Clicking on a recipe item takes you to the recipe details page `/recipes/:id/` as shown in the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true)Clicking on a recipe item takes you to the recipe details page /recipes/:id/ as shown in the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true)
- On the recipe show page, you should be able to add ingredient.
- On the recipe show page, you should be able to generate a shopping list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/general_shopping_list.png?raw=true) and url `/general_shopping_list`On the recipe show page, you should be able to generate a shopping list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/general_shopping_list.png?raw=true) and url /general_shopping_list.
- Recipe details page matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true)
- Clicking on "Add ingredient" should lead to the form that allows user to add new food, if the user is the owner of the recipe.Recipe details page matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true)
- Clicking on "Add ingredient" should lead to the form that allows user to add new food, if the user is the owner of the recipe.

Reply:
Hello @TadesseDev thank you for the review we will quickly fix that